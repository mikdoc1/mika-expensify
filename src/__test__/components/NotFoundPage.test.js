import React from 'react';
import { shallow } from 'enzyme';
import NotFoundPage from '../../components/NotFoundPage';


it('ExpensesDashboardPage should render tcomponent with 404 error', () => {
    const wrapper = shallow(<NotFoundPage/>); 
    expect(wrapper).toMatchSnapshot()  
});