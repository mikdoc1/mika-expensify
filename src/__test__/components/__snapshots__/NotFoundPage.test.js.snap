// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExpensesDashboardPage should render tcomponent with 404 error 1`] = `
<div>
  404 - 
  <Link
    to="/"
  >
    Go home
  </Link>
</div>
`;
